econml/tests/test_dowhy.py

```diff
@@ -38,18 +38,18 @@ def clf():
                   "xlearner": XLearner(models=reg(), cate_models=reg(), propensity_model=clf()),
                   "cfdml": CausalForestDML(model_y=reg(), model_t=clf(), discrete_treatment=True),
                   "orf": DROrthoForest(n_trees=10, propensity_model=clf(), model_Y=reg()),
-                  "dmlateiv": DMLATEIV(model_Y_W=reg(),
-                                       model_T_W=clf(),
-                                       model_Z_W=reg(),
-                                       discrete_treatment=True,
-                                       discrete_instrument=False)}
+                  "orthoiv": OrthoIV(model_y_xw=reg(),
```

Comment:
Would it also be worthwhile to add a DRIV estimator too?

Reply:
Sure